{
  "affected_release": null,
  "package_state": [
    {
      "product_name": "Red Hat Enterprise Linux 7",
      "fix_state": "Under investigation",
      "package_name": "python-pillow",
      "cpe": "cpe:/o:redhat:enterprise_linux:7"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Under investigation",
      "package_name": "python-pillow",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Quay 3",
      "fix_state": "Affected",
      "package_name": "quay",
      "cpe": "cpe:/a:redhat:quay:3::el7"
    }
  ],
  "threat_severity": "Important",
  "public_date": "2020-06-25T00:00:00Z",
  "bugzilla": {
    "description": "CVE-2020-10379 python-pillow: two Buffer Overflows in libImaging/TiffDecode.c",
    "id": "1852836",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1852836"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "7.5",
    "cvss3_scoring_vector": "CVSS:3.1/AV:N/AC:H/PR:L/UI:N/S:U/C:H/I:H/A:H",
    "status": "draft"
  },
  "iava": "",
  "cwe": "CWE-120",
  "statement": "While python-pillow is listed as a dependency of Red Hat Quay. It's not used by the application.",
  "acknowledgement": "",
  "name": "CVE-2020-10379",
  "document_distribution": "",
  "details": [
    "In Pillow before 6.2.3 and 7.x before 7.0.1, there are two Buffer Overflows in libImaging/TiffDecode.c."
  ],
  "references": null
}